client/lib/plans/features-list.js

```diff
@@ -681,7 +681,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY ]: {
 		getSlug: () => constants.FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
 		getTitle: () =>
-			i18n.translate( '{{strong}}Daily{{/strong}} Backups', {
+			i18n.translate( '{{strong}}Daily{{/strong}} backups', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.